snap/patches/default_data_dir.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/src/util.cpp b/src/util.cpp
+index 57b0419..dd21329 100644
+--- a/src/util.cpp
++++ b/src/util.cpp
+@@ -467,7 +467,7 @@ boost::filesystem::path GetDefaultDataDir()
+     return GetSpecialFolderPath(CSIDL_APPDATA) / "faircoin2";
+ #else
+     fs::path pathRet;
+-    char* pszHome = getenv("HOME");
++    char* pszHome = getenv("SNAP_USER_COMMON");
```

Comment:
That works, but then people won't get the welcome screen where they can select an alternate directory. What I wanted with the patch was to have ~/snap/faircoin2/common as the preselected option on that window, so when they opened the app for the first they get the explanation and can make a choice. 

![welcome_018](https://user-images.githubusercontent.com/617831/28235588-4668c69c-68ce-11e7-9653-bbe0e06805a6.png)

But if you prefer the -datadir option, I can update it, no problem.